_docker/environments/drupal-dev-local-dcp/docker-compose.yml

```diff
@@ -149,6 +148,21 @@ services:
    ports:
     - "5900"
 
+  docker_firefox:
+   image: selenium/node-firefox-debug
+   volumes:
+    - /dev/shm:/dev/shm
+    - ../../../_cucumber/tmp_downloads:/home/awestruct/developer.redhat.com/_cucumber/tmp_downloads
+   links:
+    - apache:docker
+   environment:
+    - HUB_PORT_4444_TCP_ADDR=selhub
+    - DBUS_SESSION_BUS_ADDRESS=/dev/null
+   depends_on:
+    - selhub
+   ports:
+    - "5901"
```

Comment:
I think this should be `5900` as that is the VNC port on the container from this documentation: https://hub.docker.com/r/selenium/node-firefox-debug/

Applies to all environment definitions.